packages/nameguard-js/src/utils.ts

```diff
@@ -21,3 +23,46 @@ export function isCharacter(text: string): boolean {
 export function splitCharacters(text: string): string[] {
   return [...text];
 }
+
+export function isEmojiSequence(text: string): boolean {
```

Comment:
Is it possible for us to define this param more specifically? For example, is it supposed to be a char or a grapheme or something else? Appreciate your advice :+1:

Reply:
Changed to grapheme